packaging/google-compute-engine.spec

```diff
@@ -30,6 +30,7 @@ Requires: dracut
 Requires: google-compute-engine-oslogin
 Requires: google-guest-agent
 Requires: rsyslog
+Requires: nvme-cli
```

Comment:
you need to add a line to the el6 spec in the %install section that removes the lib/udev/google_nvme_id script. otherwise  el6 RPM would contain the script but not the nvme-cli dependency.

Reply:
How does this jive with the inclusion of google_nvme_id in the 65-* rules file that requires google_nvme_id?  Is it okay to have a rule that depends on a script that doesn't exist?  Would it be better to leave the script in but have it fail due to the lack of nvme-cli?

Reply:
Updated to remove nvme-cli as asked.

Reply:
good point. this change to 65- should only be on el7+, as discussed. however i think there's a broader fix here, we can probably just prune el6 altogether here since it's now EOL. will confirm this and update

Reply:
yes, we can prune all el6 support. i will do this in a separate PR